### DISCUSSION TITLE
Object Healing occasion

### DISCUSSION THREAD

**Discussion by zhl19911203** [minio/minio#20314] [Q&A] [answered]
Is it automatically repaired when a disk fails? If it is automatic, where is the repair interval set?Because I manually deleted a file on a disk and didn't see the file automatically repaired for a long time, I want to know the time interval for this repair, or what repair command or action needs to be executed to trigger the repair action

**Comment by klauspost** [comment, marked as answer]
Full disk replacements will trigger full healing of the drive.

Objects are checked by the scanner with a reduced frequency. Based on the number of objects it can be anything between minutes and days between healing checks. If a read of the object detects defective parts it will be scheduled for healing at once.
